packages/fxa-support-panel/src/app/app.controller.spec.ts

```diff
@@ -88,6 +88,8 @@ describe('AppController', () => {
         webSubscriptions: formattedSubscriptions[MozillaSubscriptionTypes.WEB],
         playSubscriptions:
           formattedSubscriptions[MozillaSubscriptionTypes.IAP_GOOGLE],
+        appSubscriptions:
```

Comment:
Looks like we are missing the change to `app.controller.ts`, to add `AppStoreSubscriptions`

Reply:
Yeah, I realized that this morning when it said it didn't recognize it. Working on that now :) .